pkg/psmdb/tls/certmanager.go

```diff
@@ -249,7 +249,7 @@ func (c *certManagerController) Check(ctx context.Context, config *rest.Config,
 }
 
 func translateCheckError(err error) error {
-	const crdsMapping3Error = `error finding the scope of the object: failed to get restmapping: no matches for kind "Certificate" in version "cert-manager.io/v1"`
+	const crdsMapping3Error = `error finding the scope of the object: failed to get restmapping: unable to retrieve the complete list of server APIs: cert-manager.io/v1: no matches for cert-manager.io/v1, Resource=`
```

Comment:
`err = checker.Check(ctx)`

Used to return: `error finding the scope of the object: failed to get restmapping: no matches for kind "CertificateRequest" in version "cert-manager.io/v1"`